### PR TITLE
Automate:bz-2182022 [Inditex] Test duplicates at archive zone

### PR DESCRIPTION
**Pull request by viduship**
[Bug 2182022] [RGW multisite][Archive zone][Duplicate objects in the archive zone]
 ceph-qe-scripts pr : https://github.com/red-hat-storage/ceph-qe-scripts/pull/467
https://bugzilla.redhat.com/show_bug.cgi?id=2213138

https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83573390

logs: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-4AQOTL/

### CODE REVIEW

**Comment by ckulal** [comment]
Adding DNM label for now, since ceph-qe-script PR is not merged yet.
Once that PR is merged will remove DNM

File: utility/utils.py [hunk 1]
```diff
@@ -304,6 +304,95 @@ def test_sync_via_bucket_stats(primary_rgw_node, secondary_rgw_node):
...
+        f"number of buckets at primary and secondary are {total_buckets_pri} and {total_buckets_sec}"
+    )
+    deleted_buckets_arc = json.loads(
+        archive_client_node.exec_command(
+            cmd="sudo radosgw-admin bucket list | grep deleted | wc -l"
+        )[0]
+    )
+    actual_bucket_arc = total_buckets_arc - deleted_buckets_arc
+    log.info(
+        f"number of buckets at archive site is {total_buckets_arc} - {deleted_buckets_arc}"
```
Comment (ckulal) [inline utility/utils.py:344]: can use {actual_bucket_arc} in log-info

Comment (viduship) [reply utility/utils.py:344]: alright, thanks.

**Review by ckulal — COMMENTED** [review]
Looks good

**Review by anrao19 — DISMISSED** [review]

**Review by mkasturi18 — APPROVED** [review]

**Pull request merged**